sdks/python/apache_beam/transforms/core.py

```diff
@@ -279,6 +276,12 @@ def restriction_size(self, element, restriction):
     """
     return self.create_tracker(restriction).default_size()
 
+  def split_and_size(self, element, restriction):
```

Comment:
Add this "optional" method to the list on sdks/python/apache_beam/transforms/core.py:204